docs/Documentation/Software_Tools/spack.md

```diff
@@ -0,0 +1,306 @@
+
+## Installation
+
+Multiple installations of Spack can easily be kept, and each is separate from the others by virtue of the environment variable `SPACK_ROOT`. 
+All package, build, and modulefile content is kept inside the `SPACK_ROOT` path, so working with different package collections is as simple as setting `SPACK_ROOT` to the appropriate location. 
+The only exception to this orthogonality are `YAML` files in `$HOME/.spack/<platform>`.
+Installing a Spack instance is as easy as
+
+`git clone https://github.com/spack/spack.git`
+
+Once the initial Spack instance is set up, it is easy to create new ones from it through
+
+`spack clone <new_path>`
+
+`SPACK_ROOT` will obviously need to point to `<new_path>` in order to be consistent.
+
+Spack environment setup can be done by sourcing `$SPACK_ROOT/share/spack/setup-env.sh`, or by simply adding `$SPACK_ROOT/bin` to your PATH. 
+We have also created a Lua module to execute the setup-env.sh logic, so it is simple to activate one Spack environment or another through different modules. 
+The template modulefiles are available on request. 
+Two things to keep in mind are
+1.	Bash syntax has some incompatibilities with Lua string definitions, and so it is critical to escape certain patterns.
+2.	As Spack versions progress, the `setup-env.sh` script is subject to change, so it may be necessary to create a new Spack module from the newer `setup-env.sh` logic, using the escaping patterns in the existing template modulefiles as a guide.
+
+## Setting Up Compilers
+
+Spack is able to find certain compilers on its own, and will add them to your environment as it does. 
+To see which compilers your Spack collections know about, type
+
+`spack compilers`
+
+To add an existing compiler installation to your collection, point Spack to its location through
+
+`spack add compiler <path to Spack-installed compiler directory with hash in name>`
+
+The command will add to `$HOME/.spack/linux/compilers.yaml`. 
+To configure more generally, move changes to one of the lower-precedence `compilers.yaml` files (paths described below in Configuration section).
+Spack has enough facility with standard compilers (e.g., GCC, Intel, PGI, Clang) that this should be all that’s required to use the added compiler successfully.
+
+## Available Packages in Repo
+
+`spack list`  all available packages by name. Dumps repo content, so if use local repo, this should dump local package load.  
+
+`spack list <pattern>`  all available packages that have <pattern> somewhere in their name. <pattern> is simple, not regex. 
+
+`spack info <package_name>`  available versions classified as safe, preferred, or variants, as well as dependencies. Variants are important for selecting certain build features, e.g., with/without Infiniband support. 
+
+`spack versions <package_name>`  see which versions are available 
+
+
+
+## Installed packages
+
+`spack find	                 ` list all locally installed packages 
```

Comment:
Putting this in a Table may make it easier to view

Reply:
fixed